packages/terra-clinical-result/src/flowsheet-result-cell/FlowsheetResultCell.jsx

```diff
@@ -303,11 +303,33 @@ const createFlowsheetResultCellDisplay = (resultDataSet, hideUnit, numericOverfl
     });
     const displayCount = additionalResultCount + 1;
     const hasAccessoryIcons = (firstResultAttributes.comment || firstResultAttributes.modified || firstResultAttributes.unverified);
-    const additionalResultsStackDisplay = createEndAdditionalResultsStack(displayCount, additionalResultInterpretations, hasAccessoryIcons, resultKeyID);
+    const additionalResultsStack = createEndAdditionalResultsStack(displayCount, additionalResultInterpretations, hasAccessoryIcons, resultKeyID);
+
+    if (hasAccessoryIcons) {
+      const endAccessoryIcons = createEndIcons(firstResultAttributes.comment, firstResultAttributes.modified, firstResultAttributes.unverified, resultKeyID);
+      const endDisplay = (
+        <div key="EndDisplay-AdditionalResultsAndIcons" className={cx('end-display')}>
+          {additionalResultsStack}
+          {endAccessoryIcons}
+        </div>
+      );
+      compositeCell.push(endDisplay);
+      return compositeCell;
+    }
```

Comment:
It seemed like in these files they're not really using newlines to break the code up much already. So while I wanted to add new lines I didn't because it seemed to go against some standard they're using - what do you think?

Reply:
Alright, added some newlines throughout the file: [be3aea5](https://github.com/cerner/terra-clinical/pull/865/commits/be3aea592e99436de5829afb8e9a03bdeb84ccfd)

Generally tried to add newlines to create some separation for the return statements and after if statements